lesson2/Fibonacci number.py

```diff
@@ -0,0 +1,9 @@
+if __name__ == '__main__':
+    fib=[0,1]
```

Comment:
fib=[0,1] > fib = [0,1] Use whitespaces (PEP8 code style). Change it everywhere.